api/v1/accelerator_plugin.go

```diff
@@ -60,14 +70,16 @@ type RuntimeConfig struct {
 	Env         map[string]string `json:"env"`
 	Runtime     string            `json:"runtime"`
 	Options     []string          `json:"options"`
+
+	NodeSelector map[string]string `json:"node_selector,omitempty"`
```

Comment:
nit: add an example to explain the use case of this field

Reply:
This field was unnecessary; I forgot to remove it. This was my mistake, and I apologize.